ui/js/class.FormInput.js

```diff
@@ -100,7 +101,12 @@ class FormInput extends $.Component {
 			for (const child of this.children) {
 				child.toggle(isEditable)
 			}
-			this.inPlaceLabel.props.text = this.stringValue || '-'
+			this.inPlaceLabel.props.text = this.stringValue || this.props.overlayPlaceholder
+			if(this.stringValue) {
+				this.inPlaceLabel.removeCssClass('input-empty')
+			} else {
+				this.inPlaceLabel.addCssClass('input-empty')
+			}
```

Comment:
Rename css class to "empty". Use method "toggleCssClass" instead of "if else" statement